tests/w3c_tracecontext_validation_server.py

```diff
@@ -19,31 +19,37 @@
 """
 
 import json
+from os import environ
 
 import flask
 import requests
 
-from opentelemetry import trace
-from opentelemetry.ext import http_requests
-from opentelemetry.ext.wsgi import OpenTelemetryMiddleware
-from opentelemetry.sdk.trace import TracerProvider
-from opentelemetry.sdk.trace.export import (
+# FIXME This could likely be avoided by integrating this script into the
+# standard test running mechanisms.
+
+environ[
+    "OPENTELEMETRY_PYTHON_TRACER_PROVIDER"
+] = "sdk_tracer_provider"  # isort:skip
+environ[
+    "OPENTELEMETRY_PYTHON_METER_PROVIDER"
+] = "sdk_meter_provider"  # isort:skip
+
```

Comment:
could these be replaced by a call to `set_meter_provider` and `set_tracer_provider`?

Reply:
Yes, removing